LayoutTests/imported/w3c/web-platform-tests/css/css-sizing/contain-intrinsic-size/contain-intrinsic-size-030-expected.txt

```diff
@@ -1,62 +1,32 @@
 
 PASS .test 1
-FAIL .test 2 assert_equals:
-<div class="test small-contents overflow-auto cis-height" data-expected-client-width="0" data-expected-client-height="50" data-expected-scroll-width="40" data-expected-scroll-height="50"></div>
-clientHeight expected 50 but got 0
+PASS .test 2
 FAIL .test 3 assert_equals:
```

Comment:
The cases 3, 15 and 16 use `scrollbar-gutter: stable;` to turn on the scrollbar. `scrollbar-gutter` is not supported on WebKit yet, so these cases are failing now.
Left a comment on bug: 167335